source/agora/consensus/EnrollmentManager.d

```diff
@@ -164,6 +164,12 @@ public class EnrollmentManager
 
         // load next height for preimage revelation
         this.next_reveal_height = this.getNextRevealHeight();
+
+        // load the count of 'populate' of `PreImageCycle`
+        const uint populate_count =
+            cast(uint)this.loadIntegerData("cycle_count");
```

Comment:
Why not change `loadIntegerData` to return a `uint`?

Reply:
I intended that the function would be used for `getNextRevealHeight`.